assist/bin/rack/write_ontology.pl

```diff
@@ -84,21 +84,37 @@
     directory_file_path(OntologyDir, FileName, FilePath).
 
 
-%! export_thing(+Thing, -Export) is det.
+%! export_class(+Class, -Export) is det.
 %
-%    Gives the export string for a thing.
-export_thing(Thing, Export) :-
-    string_lower(Thing, LowercaseThing),
-    atomic_list_concat([LowercaseThing, '/1'], Export).
+%    Gives the export string for RDF query of a class.
```

Comment:
For export_class and export_property, you now have assert forms and the docs indicate the old are queries and the new are assertions, but you don't really say what an assertion is.  You might want to indicate that a query succeeds only if the RDF triple already exists, and the assert creates a new RDF triple (or maybe provide a reference to rdf/3 and rdf_assert/3.

Reply:
I like your suggestion, though I felt like this information would be most useful written in the created files, so that people seeing `assert_file` understand what it attempts to do.

I've pushed a commit that adds such comments in the generated files. What do you think?

Reply:
Looks good!